docs/source/core_developers_guide.rst

```diff
@@ -0,0 +1,12 @@
+
+**********************
+Core Developer's Guide
+**********************
+
+.. toctree::
+   :maxdepth: 2
+
+   core_developers_guide/sawtooth_vagrant
+   core_developers_guide/intkey_workload.rst
```

Comment:
A directory would be `core_developers_guide/sawtooth_vagrant/`. I am no expert on sphinx or RST, but I assume you can leave off the `.rst` and it will be added automatically.